src/shared/api/urls.ts

```diff
@@ -61,7 +61,7 @@ export function buildCBioPortalPageUrl(
             : pathnameOrParams;
     return URL.format({
         protocol: window.location.protocol,
-        host: AppConfig.baseUrl,
+        host: AppConfig.baseUrl || AppConfig.frontendUrl?.slice(2, -1),
```

Comment:
should we add a comment here? it's not immediately clear to me why we are slicing the frontend url.